scene/gui/rich_text_label.cpp

```diff
@@ -1501,6 +1502,9 @@ void RichTextLabel::_notification(int p_what) {
 				update();
 			}
 		} break;
+		case Control::NOTIFICATION_DRAG_END: {
```

Comment:
```suggestion
		case NOTIFICATION_DRAG_END: {
```
`NOTIFICATION_DRAG_END` is defined in Node, the `Control::` part isn't necessary.
(your previous PRs seem to have done the same)

Reply:
done